Passive_Yaw_with_N_Waves/README.md

```diff
@@ -0,0 +1,13 @@
+# Passive Yaw
+
+**Author:**  	Salman Husain
+
+**Version:** 	WEC-Sim v5.0.1 and later
+
+**Geometry**	OSWEC
+
+
+Example on using [N-Wave Spectra](http://wec-sim.github.io/WEC-Sim/advanced_features.html#NWave) with [Passive Yaw](http://wec-sim.github.io/WEC-Sim/advanced_features.html#passive-yaw-implementation) to run WEC-Sim for the [OSWEC](http://wec-sim.github.io/WEC-Sim/tutorials.html#oscillating-surge-wec-oswec) geometry. Execute the `runYawCases.m` script to run this case. 
```

Comment:
@salhus Need to remove `Execute the 'runYawCases.m' script to run this case.` If you decide to eliminate that function.

Reply:
@nathanmtom 
- done, after the latest push